server/src/bots/viber/index.ts

```diff
@@ -0,0 +1,52 @@
+import { Express } from 'express';
+import * as Viber from 'viber-bot';
+
+export async function runViberBot(
```

Comment:
API Looks super similar to telegram.
That's cool

Reply:
that's true. There are a lot things to improve, honestly. SOOOO MUCH TO MOVE TO ABSTRACT